docs/.vuepress/config.js

```diff
@@ -489,6 +477,304 @@ module.exports = {
 						path: '/extensions/themes',
 						title: 'Themes',
 					},
+					{
+						type: 'divider',
+					},
+					{
+						type: 'page',
+						path: '/extensions/endpoints',
+						title: 'Endpoints',
+					},
+					{
+						type: 'page',
+						path: '/extensions/hooks',
+						title: 'Hooks',
+					},
+					{
+						type: 'page',
+						path: '/extensions/email-templates',
+						title: 'Email Templates',
+					},
+					{
+						type: 'page',
+						path: '/extensions/migrations',
+						title: 'Migrations',
+					},
+					{
+						type: 'divider',
+					},
+					{
+						type: 'group',
```

Comment:
Indeed would be great if we could limit it to the classes actually exposed to extensions as quite a bit of extra stuff is generated 🤔 I do like "ItemsService" at the top as i suspect that would be the most used

Reply:
Does typedoc auto-generate an index page? It could be enough to just have a single "Services" link in the nav, and have the main index page take it from there

Reply:
It does in `/docs/generated/modules.md` that could definitely work but is pretty minimal

Reply:
I think that's fine! Beats having to manually keep the navigation up to date